plugins/inputs/phpfpm/phpfpm.go

```diff
@@ -58,9 +62,19 @@ var sampleConfig = `
   ##       "fcgi://10.0.0.12:9000/status"
   ##       "cgi://10.0.10.12:9001/status"
   ##
-  ## Example of multiple gathering from local socket and remove host
+  ## Example of multiple gathering from local socket and remote host
   ## urls = ["http://192.168.1.20/status", "/tmp/fpm.sock"]
   urls = ["http://localhost/status"]
+
+  ## Set response timeout for http[s] urls (default 5 seconds)
+  # response_timeout = "5s"
```

Comment:
Now that you mentioned this was based on httpjson... we should probably clean this up.  Let's do:
```
  ## Duration allowed to complete HTTP requests.
  # timeout = "5s"
```

Key features here are not explicitly saying the default, instead we rely on having it shown in the example value.  Also we use the "duration" to indicate that it is a duration string and use a complete sentence.